examples/node-app-mssql/README.md

```diff
@@ -50,3 +50,14 @@ Calculations:
 
 Current date time: 2019-08-02T09:29:24.730Z
 ```
+
+## Docker
+Set `DB_PASSWORD` (password for `SA` user) in environment. e.g 
+```bash
+$ export DB_PASSWORD=Password@123
+```
+Then
+```bash
+$ docker-compose up mssql
+$ docker-compose up app
```

Comment:
You're already defined a dependency for `mssql` in the `docker-compose.yml` file so you can now just do this:
```
$ docker-compose up app
```

Reply:
We need to run `$ docker-compose up app` only after `mssql` has started. Thus keeping this as it is.

And I removed `mssql` from `depends_on` because it would start `mssql` again even if it was running. And that caused `sync-db` script to not connect to db.

Reply:
Makes sense.